packages/docs-site/tsconfig.json

```
@@ -0,0 +1,9 @@
+{
+  "compilerOptions": {
```

Like in our other packages' tsconfigs:
```suggestion
{
  "extends": "../../tsconfig.json",
  "compilerOptions": {
```